cmd/rekor-server/app/serve.go

```diff
@@ -74,6 +76,18 @@ var serveCmd = &cobra.Command{
 			slog.Error(fmt.Sprintf("failed to initialize signer: %v", err.Error()))
 			os.Exit(1)
 		}
+		pubkey, err := signer.PublicKey()
+		if err != nil {
+			slog.Error(fmt.Sprintf("failed to get public key from signing key: %v", err.Error()))
+			os.Exit(1)
+		}
+		der, err := x509.MarshalPKIXPublicKey(pubkey)
+		if err != nil {
+			slog.Error(fmt.Sprintf("failed to marshal public key to DER: %v", err.Error()))
+			os.Exit(1)
+		}
+		slog.Info("Loaded signing key", "pub key PEM", base64.StdEncoding.EncodeToString(der))
```

Comment:
pub key PEM -> pub key base64 encoded DER ?

Reply:
My understanding is that  PEM is exactly that (surrounded by the header/footer string). 

I can look for a pem specific api to do that if it seems better though

Reply:
PEM requires the header and footer though?

Reply:
I think keeping it base64 encoded DER is fine, since thats what we do in trusted_root.json and stuff.

Reply:
Ah I see, you are saying "base64 encoded DER" would be a more useful description than "PEM".

I'm not sure I completely agree but I can change that.

Reply:
I love an excuse to look at an RFC :)

https://datatracker.ietf.org/doc/html/rfc7468#section-3, based on https://datatracker.ietf.org/doc/html/rfc1421, notes that PEM encodings begin and end with a header. 

I do agree with printing base64-encoded DER, since that’s what we need in the trusted root.